### PR TITLE
Add paste and go to context menu

### LINKED ISSUE #2273: [Feature Request]: paste and go for context or separate button
### Guidelines

- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

currently no paste and go available 

### Proposed Solution

please introduce paste and go button next to search box

### Alternatives Considered

paste and go in context menu also does the job, although it's not as smooth experience as separate button

### Issue Labels

ease of use improvement

### Additional Information

_No response_

### LINKED ISSUE #2277: [Bug]: right click on video player doesnt say Hide Video Statistics
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

1. Go to any video
2. Right click on player
3. Click on Show Video Statistics
4. Right click on player again
5. Says 'Show Video Statistics' instead of Hide Video Statistics

### Expected Behavior

Should say Hide Video Statistics when Video Statistics are enabled.

### Issue Labels

inconsistent behavior, text/string issue

### FreeTube Version

https://github.com/FreeTubeApp/FreeTube/actions/runs/2410945051

### Operating System Version

Win10 21H2

### Installation Method (If applicable)

_No response_

### Last Known Working FreeTube Version (If Any)

_No response_

### Primary API used

_No response_

### Additional Information

_No response_

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

### PR DESCRIPTION
**Pull request by vallode**
---
Add paste and go to context menu
---

**Pull Request Type**
Please select what type of pull request this is:
- [x] Feature Implementation

**Related issue**
Closes #2273
Closes #2277 

**Description**
Adds "paste and go" to context menu

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] 10
 - OS Version: [e.g. 22] Debian
 - FreeTube version: [e.g. 0.8] upstream/development

### CODE REVIEW

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/73130443/172072609-e4106b2e-c260-45d0-a925-3128d190e842.mp4

**Comment by vallode** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc can't replicate :/ I'm not even sure why this would happen, I only changed the label in line with the discussion in #2277

**Comment by vallode** [comment]
@ChunkyProgrammer @absidue if you have time could you test this out? Not sure why the above issue would happen with this PR.

**Comment by PikachuEXE** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc 
I have no problem

https://user-images.githubusercontent.com/1018543/172972677-40494489-8397-4de3-8154-56295129862a.mov

**Review by PikachuEXE — COMMENTED** [review]
I tested on search input on navbar and history/playlist tab

However I am unsure whether it should appear in input boxes in settings (and other places that's not for searching)

Screenshot 2022-06-10 at 09 38 01
Screenshot 2022-06-10 at 09 38 21
Screenshot 2022-06-10 at 09 38 28

**Review by PikachuEXE — COMMENTED** [review]
I tested on search input on navbar and history/playlist tab

However I am unsure whether it should appear in input boxes in settings (and other places that's not for searching)

Screenshot 2022-06-10 at 09 38 01
Screenshot 2022-06-10 at 09 38 21
Screenshot 2022-06-10 at 09 38 28

File: src/main/index.js [hunk 1]
```diff
@@ -21,8 +22,19 @@ function runApp() {
...
+        visible: parameters.isEditable,
+        enabled: clipboard.readText(),
+        click() {
+          const target = browserWindow.webContents || (browserWindow.id && browserWindow)
+          target.paste()
+          target.sendInputEvent({ type: 'keydown', keyCode: 'Enter' })
+        }
+      },
+      {
+        label: 'Toggle Video Statistics',
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline src/main/index.js:37]: change to [Show / Hide Video Statistics](https://github.com/FreeTubeApp/FreeTube/issues/2277#issuecomment-1146831705)

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]
I agree with @PikachuEXE it should be removed on pages where it is unnecessary to use this

like the history, playlist and settings page

but also here
Screenshot (257)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@vallode please implement the requested changes

**Comment by absidue** [comment]
I don't think this is possible to implement unless stuff changes on Electron's side and they start passing the id or the classes of the element along with the context menu event. Currently there is no way to check if a specific search box was right clicked on.

**Pull request closed without merging**